tests/test_queue_functions.py

```diff
@@ -54,6 +54,18 @@ def test_queue_message_object_return(self):
         self.assertEqual(r.status_code, 200)
         self.assertEqual(r.text, 'test-message-object-return')
 
+    def test_queue_untyped_return(self):
```

Comment:
Not fully understanding the flow of this test case. The `put_queue_untyped_return` function adds a message to the queue `testqueue-untyped-return` via an http post. After the sleep, the `get_queue_untyped_blob_return` reads from an input blob and returns an http response. The second function call does not use the queue message from the first function - is that intended? The naming and structure of the function suggests a relation between the two.

Reply:
Each test is independent to avoid races.  The flow in each trigger test is "put data" -> "wait for trigger" (trigger writes a blob) -> "check the blob to ensure trigger ran".

Reply:
I'm not able to match up the code with flow that you've described. Here's what I see as the flow for each of these requests. 

1. `put_queue_untyped_return`: POST Http data -> return queue message with bytes type. This is meant to check that we are able to write queue message with `bytes`?
2. `get_queue_untyped_blob_return`: GET request -> read blob data -> return data as string via HTTP.  The function does not make use of queues so I'm not sure why queue is mentioned in the name. The intention here is to check that HTTP messages can be returned with `str` type? 

I'm also not sure why both of these calls are in the same test function `test_queue_untyped_return` even though they are completely independent and are not using the same resources either.

Reply:
> The function does not make use of queues so I'm not sure why queue is mentioned in the name. The intention here is to check that HTTP messages can be returned with str type?

The intent is to verify that the queue trigger has fired.  There is no way to do this other than make the trigger produce a verifiable side-effect.  Writing blobs seems like a most reliable way.  The name of the function contains `queue` because it belongs logically to the same test.